File: 240403/n개의 숫자 중 최소/min-of-n-num.py
import sys

n = int(input())
arr = list(map(int, input().split()))

m = sys.maxsize
cnt = 0
for i in arr:
    if i<=m:
        if m==i:
            cnt+=1
        else :
            cnt = 1
        m=i
        
print(m, cnt)

File: 240625/n개의 숫자 중 최소/min-of-n-num.js
const fs = require("fs")
let input = fs.readFileSync(0).toString().trim().split("\n")

let n = Number(input[0]);
let arr = input[1].split(" ").map(Number);

let minV = Number.MAX_SAFE_INTEGER

let cnt = 0

for(let i=0; i<n; i++){
    if(arr[i]===minV){
        cnt++
    }
    if(arr[i]<minV){
        minV=arr[i]
        cnt = 1
    }
    
}

console.log(minV, cnt)